Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -28,13 +28,17 @@
 
 #import "APIConversions.h"
 #import "Device.h"
-#if ENABLE(WEBGPU_SWIFT)
-#import "WebGPUSwiftInternal.h"
-#endif
+
 #import <wtf/CheckedArithmetic.h>
 #import <wtf/StdLibExtras.h>
 #import <wtf/TZoneMallocInlines.h>
 
+#if ENABLE(WEBGPU_SWIFT)
+#import "WebGPUSwiftInternal.h"
+
+SWIFTCXX_THUNK(WebGPU::Buffer, copy, void, const std::span<const uint8_t>, const size_t);
```

Comment:
I would call this DEFINE_SWIFTCXX_THUNK. In case there's ever a DECLARE partner name.